projects/js-packages/publicize-components/src/components/one-click-sharing-dropdown/share-arrow-icon.tsx

```diff
@@ -0,0 +1,17 @@
+/**
+ * WordPress dependencies
+ */
+import { SVG, Path } from '@wordpress/components';
+
+const shareArrowIcon = (
```

Comment:
This can goto `js-packages/components/components/icons`

Reply:
Done